app/templates/admin/registered_users.html

```diff
@@ -37,24 +37,95 @@ <h2 class="ui header">
+                            <th rowspan="2">Primary Phone</th>
+                            <th rowspan="2">Secondary Phone</th>
+                            <th rowspan="2">Primary Language</th>
+                            <th rowspan="2">Secondary Language</th>
+                            <th rowspan="2">Requested Before?</th>
+
+                            <!--Patient Information-->
+                            <th colspan="18">Patient Information</th>
+                        </tr>
+                        <tr>
+                            <th>Patient Full Name</th>
+                            <th>Patient Birthday</th>
+                            <th>Patient Gender</th>
+                            <th>Hospital in Philadelphia</th>
+                            <th>Hospital Department</th>
+                            <th>Treatment Description</th>
+                            <th>Diagnosis</th>
+                            <th>Date of First Appointment</th>
+                            <th>Check In</th>
+                            <th>Check Out</th>
+                            <th>Treating Doctor</th>
+                            <th>Doctor Phone Number</th>
+                            <th>Hospital Social Worker</th>
+                            <th>Hospital Social Worker Phone Number</th>
+                            <th>Impatient/Outpatient</th>
+                            <th>Impatient Stay?</th>
+                            <th>Vaccinated?</th>
+                            <th>Additional Comments</th>
                         </tr>
                     </thead>
                     <tbody>
                     {% for u in users | sort(attribute='last_name') %}
-                        <tr onclick="window.location.href = '{{ url_for('admin.user_info', user_id=u.id) }}';">
-
+                    <!--modal-->
+                    <div class="ui modal">
+                        <div class="header">Confirm Deletion Request</div>
+                        <div class="content">
+                            <p>Do you want to delete this entry?</p>
+                        </div>
+                        <div class="actions">
+                            <div class="ui red approve button" onclick="delete({{ u.id }})">Confirm</div>
+                            <div class="ui cancel button">Cancel</div>
+                        </div>
+                    </div>
+                    <!--onclick="window.location.href = '{{ url_for('admin.user_info', user_id=u.id) }}';"-->
+                        <tr>
+                            <td>
+                                <button class="ui red button" id="push">Delete</button>
+                            </td>
                             <td>{{ u.first_name }}</td>
                             <td>{{ u.last_name }}</td>
                             <td>{{ u.email }}</td>
                             <td class="user role">{{ u.role.name }}</td>
+                            <td></td>
```

Comment:
Instead of empty table cells here, let's be sure that we add the room request data corresponding to the table column headers!